rust/agents/scraper/migration/src/m20220805_000005_create_table_delivered_message.rs

```diff
@@ -59,7 +59,7 @@ impl MigrationTrait for Migration {
             .create_index(
                 Index::create()
                     .table(DeliveredMessage::Table)
```

Comment:
I think in the context of an inbox it makes sense what "processed" means, but in the context of the system as a whole it is very ambiguous so I prefer the use of "delivered". I could see "processed" meaning that it was picked up by a relayer, or that it was validated as well.

If we really want to change it I would like to make it a change in my new branch rather than trying to merge the changes in.

Reply:
Yeah let's not change here, I still think changing it would be good to keep things consistent but I'm curious if anyone else feels strongly